include/pthread-barrier.h

```diff
@@ -18,7 +18,11 @@ OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 #define _UV_PTHREAD_BARRIER_
 #include <errno.h>
 #include <pthread.h>
+#if defined( __MVS__)
+#define sem_t int
```

Comment:
I have removed this. Instead I will define the following for z/OS.
#define UV_BARRIER_STRUCT_PADDING 0
For z/OS there is no need for binary compatibility with previous release because it was not enabled in the previous release :smile: